src/main/views/cookies.njk

```diff
@@ -1,5 +1,9 @@
 {% extends "layout.njk" %}
 
+{% set heading = 'Cookies' %}
+
+{% set headingVisible = false %}
```

Comment:
h1 is set in the shared component, this uses the template flag to hide it but still use it in the title